src/BaGet/Program.cs

```diff
@@ -4,63 +4,79 @@
 using McMaster.Extensions.CommandLineUtils;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
+using System.IO;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Hosting;
 
 namespace BaGet
 {
-    public class Program
+    namespace BaGet
     {
-        public static async Task Main(string[] args)
+        public class Program
         {
-            var app = new CommandLineApplication
+            public static async Task Main(string[] args)
             {
-                Name = "baget",
-                Description = "A light-weight NuGet service",
-            };
+                var app = new CommandLineApplication
+                {
+                    Name = "baget",
+                    Description = "A light-weight NuGet service",
+                };
 
-            app.HelpOption(inherited: true);
```

Comment:
Please keep the parameter name. The .NET Foundation style is to provide the parameter name for any boolean/integer literal value.

Reply:
I guess, we really need a code style checker now 😄  That wasn't really intended, just Resharper removing everything that is not necessarily needed. I will use VS2019 preview (which doesn't have any extensions enabled to avoid these kind of errors in the future 👍